app/models/MovieWithID.scala

```diff
@@ -0,0 +1,26 @@
+package models
+
+import reactivemongo.bson.BSONObjectID
+
+
+case class Movie(
+                  var image: String,
+                  var title: String,
+                  var director: String,
+                  var description: String,
+                  var actors: List[String],
+                  var screenings: List[String])
+
+case class MovieWithID(
+                        _id: BSONObjectID,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field name '_id' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5277100)